sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/components/CreateNotificationPresenter.groovy

```diff
@@ -21,10 +22,11 @@ class CreateNotificationPresenter implements CreateNotificationOutput, FetchSubs
     }
 
     @Override
-    void createdNotifications(Map<Subscriber, String> notificationPerSubscriber) {
-        notificationPerSubscriber.each {
-            this.notificationPerSubscriber.put(it.key, it.value)
-        }
+    void createdNotifications(List<NotificationContent> notifications) {
+      //TODO
+//        notificationPerSubscriber.each {
+//            this.notificationPerSubscriber.put(it.key, it.value)
```

Comment:
What are you missing here? Is this were Steffens implementation comes into play?